index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+    <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100..900;1,100..900&display=swap"
+        rel="stylesheet"
+    >
+    <link
+        rel="stylesheet"
+        href="./styles/reset.css"
+    >
+    <link
+        rel="stylesheet"
+        href="./styles/index.css"
+    >
+
+    <title>Task - HTML and CSS basics</title>
+</head>
+
+<body>
+    <header class="header row">
+        <div class="header__container container">
+            <h1 class="header__logo cell"><img
+                    src="./images/logo.svg"
+                    alt=""
+                ></h1>
+            <nav class="nav__container cell">
+                <ul class="nav">
+                    <li class="nav__item"><a
+                            class="nav__link"
+                            href="#theteam"
+                        >The Team</a></li>
+                    <li class="nav__item"><a
+                            class="nav__link"
+                            href="#pricing"
+                        >Pricing</a></li>
+                    <li class="nav__item"><a
+                            class="nav__link"
+                            href="features"
+                        >Features</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="header__container header__container-bottom container">
+
+            <article class="article__header cell">
+                <h2 class="header__heading ">Beautiful Free Nova Template</h2>
+                <p class="header__paragraph">A top notch premium quality template for your next web project.
+                </p>
+                <button class="header__btn">Free download</button>
+            </article>
+            <section class="header__image cell">
+
+                <figure class="header__image-style">
+                    <img
+                        src="./images/screen.png"
+                        alt=""
+                    >
+                </figure>
+            </section>
+        </div>
+    </header>
+    <section class="features row">
+        <header class="header__features container">
+            <h2 class="features__heading">features</h2>
+        </header>
+        <div class="features__main container">
+            <div class="feature__item feature__item-left cell">
```

Comment:
TUtaj bym zrobił `ul > li` bo mamy listę elementów, gdzie `li` to `.feature__item`